src/common/arrow/arrow_converter.cpp

```diff
@@ -276,7 +279,10 @@ void SetArrowFormat(DuckDBArrowSchemaHolder &root_holder, ArrowSchema &child, co
 		if (options.arrow_lossless_conversion) {
 			SetArrowExtension(root_holder, child, type, context);
 		} else {
-			if (options.arrow_offset_size == ArrowOffsetSize::LARGE) {
+			if (options.arrow_output_version >= 14) {
+				// Views are only introduced in arrow format v1.4
+				child.format = "vz";
```

Comment:
I was wondering if there was a `vZ` format 🤔 
But I couldn't find any mention of it here: https://arrow.apache.org/docs/format/CDataInterface.html#data-type-description-format-strings